From: Adam Williamson <awilliam@redhat.com>
Date: Tue, 17 Oct 2023 18:42:22 -0700
Subject: Replace deprecated JSONIFY_PRETTYPRINT_REGULAR usage

This was deprecated in 2.2.0, and is gone in 2.3.0. We already
require 2.2.4 or higher. The deprecation notice says to set
`app.json.compact` instead, so we'll do that!

Signed-off-by: Adam Williamson <awilliam@redhat.com>
Origin: https://github.com/psf/httpbin/pull/34
---
 httpbin/core.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/httpbin/core.py
+++ b/httpbin/core.py
@@ -89,7 +89,7 @@
 
 app = Flask(__name__, template_folder=tmpl_dir)
 app.debug = bool(os.environ.get("DEBUG"))
-app.config["JSONIFY_PRETTYPRINT_REGULAR"] = True
+app.json.compact = False
 
 app.add_template_global("HTTPBIN_TRACKING" in os.environ, name="tracking_enabled")
 
